src/adapters/controllers/post-invoke-port.js

```diff
@@ -13,6 +13,10 @@ export default function anyInvokePortFactory(invokePort) {
       const result = await invokePort({
         port: httpRequest.params.port,
         args: httpRequest.body,
+        params: httpRequest.query,
+        method: httpRequest.method,
```

Comment:
@vi-ssc (or @velosomarcus ?) fix for methods, params and headers in custom ports.